bgpd/bgp_vty.c

```diff
@@ -5034,6 +5034,41 @@ DEFUN(no_bgp_fast_convergence, no_bgp_fast_convergence_cmd,
 	return CMD_SUCCESS;
 }
 
+static int bgp_set_no_auto_ra_flag(struct vty *vty, bool flag)
+{
+	if (vty->node == CONFIG_NODE) {
+		struct listnode *node, *nnode;
+		struct bgp *bgp;
+
+		COND_FLAG(bm->flags, BM_FLAG_IPV6_NO_AUTO_RA, flag);
+		for (ALL_LIST_ELEMENTS(bm->bgp, node, nnode, bgp))
+			COND_FLAG(bgp->flags, BGP_FLAG_IPV6_NO_AUTO_RA, flag);
+	} else {
+		VTY_DECLVAR_CONTEXT(bgp, bgp);
+		COND_FLAG(bgp->flags, BGP_FLAG_IPV6_NO_AUTO_RA, flag);
+	}
+	return CMD_SUCCESS;
+}
+
+DEFUN (bgp_ipv6_auto_ra,
```

Comment:
DEFPY please, and please combine into a single DEFPY for both forms (`[no] bgp ipv6-auto-ra`).

Reply:
Done.